src/js/upcoming-this-month.js

```diff
@@ -1,6 +1,8 @@
 import TmdbApi from './tmdb-api.js';
+import Library from './local-movie-manager.js';
 
 const tmdb = new TmdbApi();
+const library = new Library();
```

Comment:
na instancji jako parametr nadaj klucz o nazwie np "local-movies"
bo tak to na undefined leci

Reply:
Ok, coś pozmieniałam.